Core1Platform/BasicPhilosophy/04_ModuleOverview.md

```diff
@@ -0,0 +1,115 @@
+# Module overview
+
+In this part of the *Actindo Core1 Platform* documentation, you will find a brief overview on the most important *Actindo Core1 Platform* modules and technical components as well as a description of the data streams between these components.   
+See the interaction between the modules also in the overview graphic [Module overview &ndash; Diagram](./04a_GraphicModuleOverview.md).
+
+
+## Basic plugins and modules
+
+The following modules and plugins are the basis for the *Actindo Core1 Platform*. Their features control and process the data flow, the data processing, the data mapping as well as the communication with third party applications. In addition, it provides master data needed for a lot of modules.
+
+**Core1 Platform**   
+The *Core1 Platform* is underlying any plugin. It offers basic functions such as the following:
+- Request handling
+- User management
+- Asynchronous jobs 
+- License handling
+- Event system
+- Database connection
+- Sandbox system   
+- and much more...
+
+**Process orchestration**    
+The *Workflow* module is a tool that allows you to shape, control, map, and monitor your business processes. Based on your special and individual business rules, you can map and made available your company's processes &ndash; from simple processes to complex workflows. You can build your own custom workflows to automate the processing of entities such as products and orders.
+
+**Currency**   
+The *Currency* plug-in provides a list of currencies with their ISO codes as well as daily exchange rates provided by the European Central Bank (ECB).
+
+**Taxes**   
+The *Taxes* module allows you to configure the automatic tax feature of the system to match your specific business requirements. You can define specific tax classes applicable to your products and determine different tax zones where you operate. Additionally, you can set up an unlimited number of tax rates, which you can also specify according to their validity period.   
+For each tax zone &frasl; tax class combination you can define tax rates. You can determine relevant tax zones based on country, ZIP-code, B2B/B2C, and much more.
+
+**Countries and languages**   
+The *Countries and languages* module <!---ist das ein Modul?--> provides any country with their ISO code and supports country classifications such as European Union (EU).
```

Comment:
Nee, ich würde sagen, es ist ein Plugin. (Es wird nicht in der linken Menü aufgelistet).